spec/features/staff/users_can_view_activities_spec.rb

```diff
@@ -0,0 +1,185 @@
+RSpec.feature "Users can view activities" do
+  context "when the user is not logged in" do
+    it "redirects the user to the root path" do
+      activity = create(:activity)
+      visit organisation_activity_path(activity.organisation, activity)
+      expect(current_path).to eq(root_path)
+    end
+  end
+
+  context "when the user is signed in as a BEIS user" do
+    let(:user) { create(:beis_user) }
+    before { authenticate!(user: user) }
+
+    scenario "the activities index shows activities associated to the BEIS organisation" do
+      activities = create_list(:programme_activity, 5, organisation: user.organisation)
```

Comment:
If we're only checking for 2 records, we might be able to save a little time by creating fewer than 5 new records?